src/execution/index/art/art.cpp

```diff
@@ -439,6 +439,40 @@ bool ART::SearchEqual(ARTIndexScanState *state, idx_t max_count, vector<row_t> &
 	return true;
 }
 
+void ART::SearchEqualJoin(Value &equal_value, vector<row_t> &result_ids) {
+	vector<row_t> row_ids;
```

Comment:
This method seems equivalent to ::SearchEqual but called with `max_count = idx_t(-1)`, perhaps use that one or merge these two methods?